bin/get-connectors-with-clusters

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-CLUSTERS=$(get-api kafka_connector_clusters | jq '.items')
-NAMESPACES=$(get-api kafka_connector_namespaces | jq '.items')
-CONNECTORS=$(get-api kafka_connectors | jq '.items')
+BASE_PATH=$(dirname -- "$( readlink -f -- "$0"; )";)
```

Comment:
Using a relative base path makes the command executable out of this direnv dirs that are outside of the rhoc bin dir.